com.valvesoftware.Steam.json

```diff
@@ -215,10 +214,6 @@
                 {
                     "type": "file",
                     "path": "com.valvesoftware.Steam.cmd.metainfo.xml"
```

Comment:
Do we still need this?

Reply:
Probably no, but it is not causing any harm.